plugin/src/main/java/io/jenkins/plugins/casc/core/HudsonPrivateSecurityRealmConfigurator.java

```diff
@@ -29,7 +29,7 @@
     private static final Logger logger = Logger.getLogger(HudsonPrivateSecurityRealmConfigurator.class.getName());
     private static final VersionNumber MIN_VERSION_FOR_HASHED_PASSWORDS = new VersionNumber("2.161");
     /// matches HudsonPrivateSecurityRealm.JBCRYPT_HEADER
-    private static final String HASHED_PASSWORD_PREFIX = "#jbcrypt:";
+    private static final String HASHED_PASSWORD_PREFIX = "#jbcrypt:"; // Noncompliant
```

Comment:
not sure why this comment is needed?

Reply:
Sonarqube will detect it, and report it as a security issue due to keyword `PASSWORD`.

Reply:
would be great to have sonarcloud setup 😉

Reply:
Agree. It'll be better if there're more automatic-check exists.